### PR TITLE
Fixes in packaging

### PR DESCRIPTION
**Pull request by ignatenkobrain**

### CODE REVIEW

File: dnf-plugins-core.spec [hunk 1]
```diff
@@ -140,10 +140,8 @@ PYTHONPATH=./plugins nosetests-%{python3_version} -s tests/
 %license COPYING
 %doc AUTHORS README.rst
 %ghost %{_var}/cache/dnf/packages.db
-%config %{_sysconfdir}/dnf/plugins/*
-%exclude %{python3_sitelib}/dnf-plugins/__pycache__/
-%exclude %{python3_sitelib}/dnf-plugins/kickstart.py
-%{python3_sitelib}/dnf-plugins/*
+%config(noreplace) %{_sysconfdir}/dnf/plugins/*
+%{python3_sitelib}/dnf-plugins/*.py
```
Comment (Conan-Kudo) [inline dnf-plugins-core.spec]: Why is this necessary over `%{python3_sitelib/dnf-plugins/*`?

Comment (ignatenkobrain) [inline dnf-plugins-core.spec]: otherwise it will own `__pycache__` directory

**Comment by ignatenkobrain** [comment]
okay, doing self-review. @dnf-bot r+
